GVFS/GVFS.FunctionalTests/Settings.cs

```diff
@@ -30,23 +30,23 @@ public static void Initialize()
                 RepoToClone = @"https://gvfs.visualstudio.com/ci/_git/ForTests";
                 Commitish = @"FunctionalTests/20180214";
 
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
-                    EnlistmentRoot = @"C:\Repos\GVFSFunctionalTests\enlistment";
-                    PathToGVFS = @"GVFS.exe";
-                    PathToGit = @"C:\Program Files\Git\cmd\git.exe";
+                    EnlistmentRoot = @"C:\Repos\GVFSFunctionalTests\enlistment";
+                    PathToGVFS = @"GVFS.exe";
+                    PathToGit = @"C:\Program Files\Git\cmd\git.exe";
                     PathToBash = @"C:\Program Files\Git\bin\bash.exe";
-
+
                     ControlGitRepoRoot = @"C:\Repos\GVFSFunctionalTests\ControlRepo";
-                    FastFetchBaseRoot = @"C:\Repos\GVFSFunctionalTests\FastFetch";
+                    FastFetchBaseRoot = @"C:\Repos\GVFSFunctionalTests\FastFetch";
                     FastFetchRoot = Path.Combine(FastFetchBaseRoot, "test");
-                    FastFetchControl = Path.Combine(FastFetchBaseRoot, "control");
-                    PathToGVFSService = @"GVFS.Service.exe";
-                    BinaryFileNameExtension = ".exe";
-                }
-                else
+                    FastFetchControl = Path.Combine(FastFetchBaseRoot, "control");
+                    PathToGVFSService = @"GVFS.Service.exe";
+                    BinaryFileNameExtension = ".exe";
+                }
+                else
                 {
-                    string root = "/GVFS.FT";
+                    string root = "~/GVFS.FT";

```

Comment:
What does `Path.Combine` return when the path includes `~`?

I ask because when we updated the default `.gvfsCache` location to be in `~` we had to use `Environment.GetEnvironmentVariable("HOME")` because the expansion of `~` was bash specific.

https://github.com/microsoft/VFSForGit/blob/fa70093b0795f6dac167e8e361f2b3249e06d1de/GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs#L219